recipes/holopy/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "HoloPy" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "fecfcdd3fb46de5b03db7ee53fa9ef6463eb75a1889143578b51707ecaeb878e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://github.com/manoharan-lab/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k or win or np<112]
```

Comment:
Our package only runs with python 3.

We cannot figure out how to compile Fortran on Windows.

Something happened when xarray upgraded to 0.9 that interferes with how we call out to our Fortran code for numpy 1.11. The simplest fix was to require users to have numpy >=1.12.